.github/workflows/deploy.yml

```diff
@@ -224,10 +224,11 @@ jobs:
       iam_path: ${{ secrets.IAM_PATH }}
       full_iam_permissions_boundary_policy: ${{ secrets.FULL_IAM_PERMISSIONS_BOUNDARY_POLICY }}
       react_app_auth_mode: AWS_COGNITO
+      nr_license_key: ${{ secrets.NR_LICENSE_KEY }}
 
   finishing-prep:
     name: Finishing Prep
-    needs: [deploy-infra, begin-deployment]
+    needs: [deploy-infra, begin-deployment, deploy-app]
```

Comment:
I was hitting an issue where `Finishing Deploy` was running before unit tests and `app-api` completed deploying, as the infra had been previously deployed and skipped. It was then adding the cypress users, which seems like it was messing up the migration task. Adding this here to wait on app deployment before `Finishing Deploy` begins.